src/components/chat-item/chat-item-card.ts

```diff
@@ -299,6 +301,101 @@ export class ChatItemCard {
       this.card?.render.insertChild('beforeend', this.fileTreeWrapper.render);
     }
 
+    /**
```

Comment:
Comment doesn't match with the content below.

Reply:
Fixed